plugin/manager.go

```diff
@@ -83,25 +87,8 @@ type controller struct {
 	timeoutInSecs int
 }
 
-// pluginRegistryService ensures that all resolved repositories
-// are of the plugin class.
-type pluginRegistryService struct {
-	registry.Service
-}
-
-func (s pluginRegistryService) ResolveRepository(name reference.Named) (repoInfo *registry.RepositoryInfo, err error) {
-	repoInfo, err = s.Service.ResolveRepository(name)
-	if repoInfo != nil {
-		repoInfo.Class = "plugin"
```

Comment:
Is this being set somewhere else?

Reply:
That's what I was wondering; this code didn't appear to be called though, so not by this code

Reply:
I could double check (not near a computer), but at least couldn't find where it was called

Reply:
Think that could relate to the failures on https://github.com/moby/moby/pull/42889 ?